lib/auth/auth_test.go

```diff
@@ -773,6 +771,8 @@ func TestLocalControlStream(t *testing.T) {
 	}
 }
 
+// Deprecated: REMOVE IN 14.0.0 along with GenerateToken. This test has been
+// superseded.
```

Comment:
nit: similarly as above, "deprecating" a test doesn't mean much I think.

I suggest either a "DELETE IN" comment, but I think we can drop the comment altogether - this has to be addressed whenever GenerateToken is removed anyway.

```suggestion
```